opentech/apply/activity/messaging.py

```diff
@@ -309,15 +328,17 @@ def batch_recipients(self, message_type, submissions, **kwargs):
         ]
 
     def reviewers_updated(self, submission, link, user, added=list(), removed=list(), **kwargs):
-        message = f'{user} has updated the reviewers on <{link}|{submission.title}>. '
+        message = [f'{user} has updated the reviewers on <{link}|{submission.title}>']
 
         if added:
-            message += 'Reviewers added: ' + ', '.join([str(reviewer.reviewer) + ' as ' + str(reviewer.role) for reviewer in added]) + '. '
+            message.append('Added:')
+            message.extend(reviewers_message(added))
 
         if removed:
-            message += 'Reviewers removed: ' + ', '.join([str(reviewer.reviewer) + ' as ' + str(reviewer.role) for reviewer in removed]) + '. '
+            message.append('Removed:')
+            message.extend(reviewers_message(removed))
 
-        return message
+        return '. '.join(message)
```

Comment:
If we look at `message` here in the breakpoint, it equals:
['Erin Super Django has updated the reviewers on <http://apply.localhost:8000/apply/submissions/3595/|Review Test 7>', 'Added:', 'Fiona Krakenbürger as Primary.', 'Tara Tarakiyee as Secondary.', 'Removed:', 'Adam Lynn as Primary.', 'Wei Fan as Secondary.']

This code is adding too many periods. We really only need one at the end of line 331 above, then we can join with a space.

Reply:
:+1: good spot.